tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_login_bzlite.py

```diff
@@ -0,0 +1,13 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.bzlite.app import Bzlite
+
+
+class TestBugzillaLite(GaiaTestCase):
+    def test_login_BugzillaLite(self):
+        self.bzlite = Bzlite(self.marionette)
+        slef.bzlite.launch()
+        # enter the username and the password
+        self.bzlite.login('manel.rhaiem92@gmail.com', '*****************')
```

Comment:
I can now reveal the trick we use to hide the passwords: We put the credentials in the testvars.json file. Then we use them [like this](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/test_dialer_receive_call.py#L27).

In your case, you can add:
```js
"bugzilla": {
    "user": "YourUser",
    "password": "YourPassword"
}
``` 
And then we can use these variables in the test:
```python
self.bzlite.login(self.testvars['bugzilla']['user'], self.testvars['bugzilla']['password'])
```